core/src/models/Export.ts

```diff
@@ -204,12 +204,15 @@ export class Export extends CommonModel<Export> {
     return this.save();
   }
 
-  async retry(retryDelay: number = config.tasks.timeout) {
+  async retry(
+    retryDelay: number = config.tasks.timeout,
+    skipCount: boolean = false
+  ) {
     const maxExportAttempts = parseInt(
       (await plugin.readSetting("core", "exports-max-retries-count")).value
     );
 
-    this.retryCount++;
+    if (!skipCount) this.retryCount++;
```

Comment:
Parallelism was making this `retry` but that's not really an error, so don't increment the count.